### DISCUSSION TITLE
One-time sponsoring

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [Roboroads/laravel-tinker#336] [Everything Else]
I really love your Tinker plugin and I would love to share it in our Laravel NL/BE Discord community (~200 members). Can a user also sponsor a one-time amount to obtain a key?

**Comment by Roboroads** [comment]
Zeker! En deze community moet ik ook maar even lid van worden 😏 Message ID: ***@***.***>

**Reply by SebastiaanKloos** [reply]
Nice! https://discord.gg/G7uW8XaZ

**Comment by Roboroads** [comment]
Answer in english, for the non-dutchies: yes. A one-time sponsor can contact me for a key as well. I'm not doing this for the money anyway ^^
